### DISCUSSION TITLE
containers stuck at wait_start

### DISCUSSION THREAD

**Discussion by alexandruhera** [Security-Onion-Solutions/securityonion#9477] [General]
Hi everyone, 

This occurs to me from time to time, the container status gets stuck at wait_start? Is there a way to address this problem?
![Screenshot 2022-12-27 142359](https://user-images.githubusercontent.com/12063284/209666292-8a43b1f7-c9da-4201-8d19-944d5b20f940.png)

**Comment by dougburks** [comment]
That usually means that you have a configuration problem somewhere. I'd recommend going back through your recent changes and double-checking for syntax errors.

**Comment by FrankyDole** [comment]
I have the exact same problem.... Don't know what to do... :s

**Comment by FrankyDole** [comment]
FOUND IT!

You see every service that is at WAIT_START ?

Do that command for everyone (starting, for example, with the first in the list, which is "curator")

sudo so-start curator

That's it!
